packages/andromeda-non-fungible-tokens/src/crowdfund.rs

```diff
@@ -68,11 +78,17 @@ pub struct CampaignConfig {
 }
 
 impl CampaignConfig {
-    pub fn validate(&self, deps: Deps) -> Result<(), ContractError> {
+    pub fn validate(&self, deps: DepsMut, env: &Env) -> Result<(), ContractError> {
         // validate addresses
         self.tier_address.validate(deps.api)?;
-        self.withdrawal_recipient.validate(&deps)?;
-        validate_denom(deps, self.denom.clone())?;
+        self.withdrawal_recipient.validate(&deps.as_ref())?;
+        // let _ = get_verified_asset();
+        let _ = self
+            .denom
+            .get_verified_asset(deps, env.clone())
+            .map_err(|_| ContractError::InvalidAsset {
+                asset: self.denom.to_string(),
+            })?;
```

Comment:
Why are we overwriting `get_verified_asset`'s error?

Reply:
For Cw20 asset, it throws generic `StdError` (which is not readable) in case the cw20 contract does not exist. Thus it is more readable to throw `InvalidAsset` error.